kolibri/deployment/default/settings/base.py

```diff
@@ -126,25 +128,38 @@
 # Database
 # https://docs.djangoproject.com/en/1.9/ref/settings/#databases
 
-DATABASES = {
-    'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(KOLIBRI_HOME, 'db.sqlite3'),
-        'OPTIONS': {
-            'timeout': 100,
+if OPTIONS.get("DATABASE_ENGINE") == "sqlite":
+    DATABASES = {
+        'default': {
+            'ENGINE': 'django.db.backends.sqlite3',
+            'NAME': os.path.join(KOLIBRI_HOME, OPTIONS.get('DATABASE_NAME') or 'db.sqlite3'),
+            'OPTIONS': {
+                'timeout': 100,
+            }
         }
-    },
-}
+    }
+elif OPTIONS.get("DATABASE_ENGINE") == "postgres":
+    DATABASES = {
+        'default': {
+            'ENGINE': 'django.db.backends.postgresql',
+            'NAME': OPTIONS['DATABASE_NAME'],
+            'PASSWORD': OPTIONS.get('DATABASE_PASSWORD') or '',
+            'USER': OPTIONS.get('DATABASE_USER') or '',
+            'HOST': OPTIONS.get('DATABASE_HOST') or '',
+        }
+    }
 
 # Content directories and URLs for channel metadata and content files
 
+_content_root = os.path.join(KOLIBRI_HOME, OPTIONS.get('CONTENT_ROOT') or 'content')
```

Comment:
Sorry if I'm not getting it right - Is the CONTENT_ROOT variable specifically for the `contentdirectory` command?

Reply:
In the above line, there are defaults being specified outside of `kolibri.utils.conf` -- perhaps better to just specify in one place.

Reply:
>Is the CONTENT_ROOT variable specifically for the contentdirectory command

Partially, but not solely. It's the option that the user can modify to change where Kolibri looks for content. It will also be used by your management command.

>In the above line, there are defaults being specified outside of kolibri.utils.conf -- perhaps better to just specify in one place.

In an update to this PR that I'm going to push shortly, (almost) all defaults and validation etc will be done in the `options` module.